portfolio/src/main/webapp/script.js

```diff
@@ -13,34 +13,28 @@
 // limitations under the License.
 
 
-/** @return {string} */
-function generateRandomGreeting(){
+/**
+ * @return {string} 
+ * a random greeting
```

Comment:
nit: we usually prefer function comments to be in the form where they sound as if they are preceded by an invisible "this function..." 

For example, instead of "a random greeting", something like "Generates a random greeting"

Reply:
Thanks for the comment. I will make the necessary adjustments